mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+    description: |
+      Service acts as a top-level container that manages a set of configurations and revision templates which implement a network service. Service exists to provide a singular abstraction which can be access controlled, reasoned about, and which encapsulates software lifecycle decisions such as rollout policy and team resource ownership.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - True
+          - False
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: The location of the cloud run service
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          Name of the Service.
+        pattern: projects/{{project}}/locations/{{location}}/services/{{name}}
+      - !ruby/object:Api::Type::String
+        name: "description"
+        description: |
+          User-provided description of the Service. This field currently has a 512-character limit.
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          Server assigned unique identifier for the trigger. The value is a UUID4 string and guaranteed to remain unchanged until the resource is deleted.
+      - !ruby/object:Api::Type::String
+        name: "generation"
+        output: true
+        description: |
+          A number that monotonically increases every time the user modifies the desired state. Please note that unlike v1, this is an int64 value. As with most Google APIs, its JSON representation will be a string instead of an integer.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: |-
+          Map of string keys and values that can be used to organize and categorize objects. User-provided labels are shared with Google's billing system, so they can be used to filter, or break down billing charges by team, component, environment, state, etc. For more information, visit https://cloud.google.com/resource-manager/docs/creating-managing-labels or https://cloud.google.com/run/docs/configuring/labels Cloud Run will populate some labels with 'run.googleapis.com' or 'serving.knative.dev' namespaces. Those labels are read-only, and user changes will not be preserved.
+      # Block on b/244872932 for implementation    
+      # - !ruby/object:Api::Type::KeyValuePairs
```

Comment:
`annotation` is currently blocked on b/244872932. Discussed with the service team and decide to omit the field from implementation until the API validation is launched.